test/agent/posts.js

```diff
@@ -27,6 +28,45 @@ scenario('Can create a post', async (s, t) => {
       base: testPost.base
     })
     console.log(get_posts_result.Ok)
-    t.deepEqual(get_posts_result.Ok, [{ ...testPost, creator: alice.info('app').agentAddress, address }], "Could retrieve the added post from the base")
+    t.deepEqual(get_posts_result.Ok, {posts: [{ ...testPost, creator: alice.info('app').agentAddress, address }], more: false}, "Could retrieve the added post from the base")
+  })
+
+scenario('Can create multiple posts and paginate', async (s, t) => {
```

Comment:
@lorenjohnson check out here for the minor changes needed to the UI code for the new return type (and for how to use the pagination)